apps/native/app/src/graphql/fragments/document.fragment.graphql

```diff
@@ -1,15 +1,20 @@
 directive @client on FIELD
 
-fragment ListDocument on Document {
+fragment ListDocument on DocumentV2 {
   id
   subject
-  senderName
-  senderNatReg
-  date
-  fileType
-  url
+  publicationDate
+  downloadUrl
   opened
   categoryId
   bookmarked
+  sender {
+    id
+    name
+  }
+  content {
+    type
+    value
+  }
```

Comment:
Should this be here?

Reply:
Yes, we are using this when fragment both when fetching the list and also when fetching document details that should include this, also hopefully one day we will get the type in the list as well 😊

Reply:
But I could exclude this from the list call and just add it for fetching the whole document.